sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/TrackNotImplementedException.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Tests.Diagnostics
```

Comment:
Fine with that (as long as you add tests).

Should we mention that exception in RSPEC text?